schema.graphql

```diff
@@ -107,6 +121,162 @@ type UnprocessedEntity @entity {
   block: Block
 }
 
+type Service @entity {
+  id: ID!
+  name: String!
+  computeUnitsPerRelay: BigInt!
+  owner: Account!
+  addServiceMsgs: [AddServiceMsg] @derivedFrom(field: "service")
+}
+
+type AppStakeMsg @entity {
+  id: ID!
+  stake: Coin!
```

Comment:
Please add the `message: Message!` relation. In general every message event should have a relation to its primitives (i.e. `Message`, `Transaction`, `Block`). Event types should also have their corresponding `Event` primitive.
```suggestion
  stake: Coin!
  message: Message!
```